docs/src/user-guide/configuring/ignoring-code.md

```diff
@@ -9,6 +9,12 @@ eleventyNavigation:
 
 ---
 
+You can configure ESLint to ignore certain files and file patterns when linting.
```

Comment:
I'm not sure if it is correct to say "to ignore file patterns". Files are ignored, file patterns are a way to specify which files are ignored.

Reply:
Agree, maybe something like this?

```suggestion
You can configure ESLint to ignore certain files and directories while linting by specifying one or more glob patterns.
```

Reply:
This looks better 👍🏻